src/Notes/MarketingNotifications.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Marketing notifications for merchants (admin).
+ *
+ * @package Pinterest_For_WooCommerce/Classes/
+ * @since   x.x.x
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Pinterest\Notes;
+use Automattic\WooCommerce\Pinterest\Notes\Collection\EnableCatalogSync;
```

Comment:
Coupled with the PHPCS error here, we should also add one more import.

```suggestion
namespace Automattic\WooCommerce\Pinterest\Notes;

use Automattic\WooCommerce\Pinterest\Notes\Collection\AbstractNote;
use Automattic\WooCommerce\Pinterest\Notes\Collection\EnableCatalogSync;
```